pennylane/interfaces/torch.py

```diff
@@ -12,191 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-.. _torch_qnode:
+interfaces.torch
+================
 
-PyTorch interface
-*****************
-
-**Module name:** :mod:`pennylane.interfaces.torch`
+**Module name**: pennylane.interfaces.torch
 
 .. currentmodule:: pennylane.interfaces.torch
 
+This module contains the :func:`TorchQNode` function to convert Numpy-interfacing quantum nodes with PyTorch
+compatible quantum nodes.
```

Comment:
This module contains the :func:`TorchQNode` function that converts Numpy-interfacing quantum nodes to PyTorch
compatible quantum nodes.